Dockerfile.stable

```diff
@@ -20,15 +24,20 @@ RUN mkdir -p /root/chrome-driver &&\
 ADD ./xstartup.sh /xstartup.sh
 RUN chmod 775 /xstartup.sh
 # selenium
-ADD http://selenium-release.storage.googleapis.com/$SELENIUM_VERSION/selenium-server-standalone-$SELENIUM_VERSION.0.jar \
+# TODO: make better url scheme
+ADD http://selenium-release.storage.googleapis.com/3.12/selenium-server-standalone-$SELENIUM_VERSION.jar \
     /root/selenium-server/selenium-server-standalone.jar
+
 # firefox
-ADD https://download-installer.cdn.mozilla.net/pub/firefox/releases/$FIREFOX_VERSION.0esr/linux-x86_64/en-US/firefox-$FIREFOX_VERSION.0esr.tar.bz2 \
+ADD https://download-installer.cdn.mozilla.net/pub/firefox/releases/$FIREFOX_VERSION/linux-x86_64/en-US/firefox-$FIREFOX_VERSION.tar.bz2 \
     /root/firefox.tar.bz2
+ADD https://github.com/mozilla/geckodriver/releases/download/$GECKODRIVER_VERSION/geckodriver-$GECKODRIVER_VERSION-linux64.tar.gz \
+    /root/gecko.tar.gz
+
 RUN tar -C /root/ -xjvf /root/firefox.tar.bz2 && rm -f /root/firefox.tar.bz2
+RUN tar -C /root/firefox/ -xvf /root/gecko.tar.gz && rm -f /root/gecko.tar.gz
 
 # runtime
-EXPOSE 22
```

Comment:
No more SSH?

Reply:
wasn't started to begin with, so this lowers port cost